appium/tests/specs/mock/inbox/CheckArchiveThreadTooAggressive.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import { MockUserList } from 'api-mocks/mock-data';
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen
+} from '../../../screenobjects/all-screens';
+
+describe('INBOX: ', () => {
+
+  it('check "archive thread" too aggressive', async () => {
+    const mockApi = new MockApi();
+
+    const testMessage = 'Test "archive thread" too agreesive';
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [testMessage],
+    });
+    mockApi.attesterConfig = {
+      servedPubkeys: {
+        [MockUserList.e2e.email]: MockUserList.e2e.pub!
+      }
+    };
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      await browser.pause(1000000);
+      await MailFolderScreen.clickOnEmailBySubject('Signed and encrypted message');
+      await EmailScreen.clickArchiveButton();
```

Comment:
@sosnovsky I need to mock new message arrive event here.
As you can see from this PR, test thread has 3 messages.
But I wanna gmail mock api to return thread with 2 messages first and then return 3 messages after user clicks `archive`.
This is to test `archive thread` too aggressive feature.

But not sure how to achieve this.
You have better understanding of how mock api works.
Do you have any suggestion on this?

Reply:
@ioanmo226 You can change messages list by calling `mockApi.addGoogleAccount` with updated list:
```swift
mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
   messages: [testMessage, testMessage2, testMessage3],
});
```

You can also rename `addGoogleAccount` to  `configGoogleAccount`, so it'll better describe functionality.

Reply:
But in this case changes for existing messages will be reset (`inbox` and `sent` labels will be added back for archived messages). It'll be better to add separate method for adding new messages, I'll work on it

Reply:
Got it. Thank you

Reply:
Please let me know once done. So that I can continue work on this test.

Reply:
@ioanmo226 I added `addGoogleMessage` mock method - https://github.com/FlowCrypt/flowcrypt-ios/pull/1769/commits/25157192684be8fe0e0aa212083c7a4016be5f71

You can use it like this
```js
mockApi.addGoogleMessage('e2e.enterprise.test@flowcrypt.com', addedMessageSubject);
```

Reply:
Got it. Thanks